data-raw/healthy-lives/behavioural-risk-factors/drug-misuse.R

```diff
@@ -0,0 +1,52 @@
+# ---- Load packages ----
+library(tidyverse)
+library(readxl)
+library(httr)
+
+# ---- Scrape URL for drug deaths ----
+# ---- Source: https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/drugmisusedeathsbylocalauthority ----
+drug_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/drugmisusedeathsbylocalauthority/current/2022localauthorities.xlsx"
+
+# ---- Download and read URL as temp file ----
+drug_temp_file <- tempfile(fileext = ".xlsx")
+download.file(drug_url, drug_temp_file, mode = "wb")
+drug_data <- read_excel(drug_temp_file)
```

Comment:
What is the purpose of this line? Doesn't look like you use it anywhere else in the script

Reply:
The purpose of the first line is to create a path for the temp file, the second line is to download the excel file and save it to the path created by the previous line, and the third line is to read the downloaded temp file into R. I think I need all three lines?

Reply:
Do you use `drug_data` elsewhere in the script? Try to comment out line 13 and see if your script runs.